### ISSUE TITLE
Group Chat (45 People) Not Being Able To Send Messages In The Group

### ISSUE THREAD

**Issue by priceless819** [signalapp/Signal-Android#10859]
- [ x] I have searched open and closed issues for duplicates
- [x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
A Group Chat of 45 people, messages are not being sent in a timely order. Several people have reported that when they send a message, the sending icon appears, and its never sent, Or sometimes its received 2 days later. 

### Steps to reproduce
- Removed the person with the problem and then added them back - same result
- Removed the Person, they deleted the chat history, then added them back - same result

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)

### Device info

**Device:** Samsung Note 20 Ultra
**Android version:** 11
**Signal version:** 5.3.9

### Link to debug log

My Debug Log:
https://debuglogs.org/69f10c938f2c394650d768524bc079ac703ac2ecab46cfa9b60a203664d6fd70

Members Debug Log With The Issue:
https://debuglogs.org/ff9a9792a416b1656e2294b2a2eeaa8a197d278f0930065f1985d351eb8ecdb2

**Comment by maj-tom** [comment]
I'm seeing the same issue with a (new type) group of 75 members I created. The group info does _not_ show any legacy group Signal version members.

Initially, about a week ago a member (A) messaged me directly that his messages weren't going through into the group, but pending indefinitely. I added the member to a new group I created with just a few people, and he was able to send without a problem. He said that in the large group, he would sometimes eventually see his message get sent to most members except two specific ones (let's call them C and D). When he blocked those two members, he was able to send messages fine.

Today, two other members (E and F) messaged me directly saying their messages are not going through to the 75-person group. I added them to the small group and they can send there. Their messages show indefinitely pending for all group members, not just for A and B as in the previous case. I removed and re-added E and F to the group, and they also tried a clean reinstall of Signal. Additionally, I removed from the group the two members A and B mentioned above, in case there is something weird with their Signal app causing the problem. When I did this removal, a message suddenly came through from yet another member (G) which was dated from yesterday. None of these resolved the issue for E and F, however.

I've confirmed with A, E, and F that they have the newest signal (some Android, some iOS). I'm guessing there are other members in the group that are not able to send either but haven't let me know yet. This is a serious issue. It's quite disappointing that no one has commented on the original post here for a week...

This is a serious issue.

**Comment by priceless819** [comment]
This was the exact problem I had. What I did was migrate everyone to a new group.  When I added people to the new group, I asked them to send 3 messages in a row, because I saw in the old group a message would go through,  then the next message would go to pending. I'm thinking this has something to do with when the servers went down and something didn't reset properly.  I have all the people in the new group and it appears to be smooth again.
